powermand/client.c

```diff
@@ -676,7 +677,7 @@ static void log_state_change(Client *c)
         return;
 
     hosts = _xhostlist_ranged_string(c->cmd->hl);
-    syslog(LOG_DAEMON | LOG_DEBUG, "%s %s%s", action, hosts,
+    syslog(LOG_DAEMON | level, "%s %s%s", action, hosts,
```

Comment:
Ooops, preexisting condition, but would you mind dropping LOG_DAEMON here?  (See previous comment).

Reply:
Actually, only pre-existing in the sense that I introduced it in an earlier commit :).
Fixed!

Reply:
Oh duhhh, I should pull my head out :-)